keyboards/formy46/rules.mk

```
@@ -0,0 +1,34 @@
+# MCU name
+MCU  =  atmega32u4
+
+# Bootloader selection
+BOOTLOADER  =  caterina
+
+# Build Options
+BOOTMAGIC_ENABLE        =  no         # Virtual DIP switch configuration
+MOUSEKEY_ENABLE         =  no        # Mouse keys
```

You have this defined twice.

```suggestion
MOUSEKEY_ENABLE         =  yes       # Mouse keys
```